lnwallet/interface.go

```diff
@@ -458,7 +459,8 @@ type WalletController interface {
 	// the selected/validated inputs. It is in the caller's responsibility
 	// to lock the inputs before handing them out.
 	FundPsbt(packet *psbt.Packet, minConfs int32,
-		feeRate chainfee.SatPerKWeight, account string) (int32, error)
+		feeRate chainfee.SatPerKWeight, account string,
+		changeScope *waddrmgr.KeyScope) (int32, error)
```

Comment:
Why did you elect to use functional options to pass the scope in the btcwallet PR but not this one? Seems func options here as well could help clarify the back and forth comments about default scope and not break the interface.

Reply:
For btcsuite, I used functional options to be consistent with some previous commit: https://github.com/btcsuite/btcwallet/commit/d00bc8369816f84bd762c936ecab59ce9f1545bf

For lnd, I did the same, I took example of some previous modifications.

But I think that, it might be a good point

Reply:
for now i’ll go with regular arguments